commands/setup.go

```diff
@@ -34,13 +34,16 @@ import (
 
 var log = logging.Logger("visor")
 
+// RunContext wraps lens and DB connections.
 type RunContext struct {
-	opener lens.APIOpener
-	closer lens.APICloser
-	db     *storage.Database
+	Opener lens.APIOpener
+	Closer lens.APICloser
+	DB     *storage.Database
 }
 
-func setupStorageAndAPI(cctx *cli.Context) (context.Context, *RunContext, error) {
+// SetupStorageAndAPI setups of the sentinel database and returns a
+// ready-to-use RunContext with Openers and Closers using the chosen lens.
+func SetupStorageAndAPI(cctx *cli.Context) (context.Context, *RunContext, error) {
```

Comment:
Purpose is that we can call SetupStorageAndAPI from a different program easily and then be able to handle DB closing from there.